agent/php_internal_instrument.c

```diff
@@ -1527,10 +1560,66 @@ NR_INNER_WRAPPER(mysqli_stmt_prepare) {
  */
 NR_INNER_WRAPPER(memcache_function) {
   nr_php_instrument_datastore_operation_call(nr_wrapper, NR_DATASTORE_MEMCACHE,
-                                             nr_wrapper->extra, NULL,
+                                             nr_wrapper->extra, NULL, false,
                                              INTERNAL_FUNCTION_PARAM_PASSTHRU);
 }
 
+NR_INNER_WRAPPER(memcached_connect_function) {
```

Comment:
This function has nothing to do with any type of connection, it is a wrapper for [`Memcached::addServer`](https://www.php.net/manual/en/memcached.addserver.php). I suggest keeping the name close to what it instruments:
```suggestion
NR_INNER_WRAPPER(memcached_add_server) {
```

Reply:
renamed in [bc833b7](https://github.com/newrelic/newrelic-php-agent/pull/958/commits/bc833b7c0cda48d97bccc0dfd2a26ec622502253)